Last-Update: 2014-04-02
Forwarded: no
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: look for libpython2.7 (fixes FTBFS in presence of python3).

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -59,8 +59,11 @@
 if (USE_BUNDLED_MYSQLDUMP)
   set(USE_BUNDLED_MYSQLDUMP -DHAVE_BUNDLED_MYSQLDUMP=1)
 endif ()
 
+set(Python_ADDITIONAL_VERSIONS "2.7")
+#include(FindPythonLibs)
+
 # In Ubuntu 13.04, the built-in check for python fails.. so we have to resort to the check with pkg-config (which doesn't work in many distros)
 find_package(PythonLibs 2.6)
 if (NOT PYTHONLIBS_FOUND)
     pkg_check_modules(PYTHON REQUIRED python)
